README.md

```diff
@@ -154,19 +209,44 @@ The command line utility for Zim:
   * Want to upgrade `zimfw` to its latest version? Run `zimfw upgrade`.
   * For more information about the `zimfw` utility, run `zimfw help`.
 
-Settings
---------
+FAQ
+----
+
+### 1. How can I change default `zimrc` file location?
+
+The location of `.zimrc` file is controlled by `ZIM_HOME` environment variable. Change it to the desired path.
+Eg: Set this in your `zshrc` to change path of `zimrc` to `~/.config/zsh/zim`:
+`export ZIM_HOME="${HOME}/.config/zsh/zim"`
+
+### 2. How can I speed up module installation?
 
 Modules are installed using `git` by default. If you don't have `git`
 installed, or if you want to take advantage of our degit tool for faster and
 lighter module installations, you can set degit as the default tool with:
 
     zstyle ':zim:zmodule' use 'degit'
 
+### 3. How to disabel automatic updates?
+
 By default, `zimfw` will check if it has a new version available every 30 days.
 This can be disabled with:
 
     zstyle ':zim' disable-version-check yes
+    
+### 4. When should I call `compinit` in my `zshrc` ?
+
+`compinit` is called by _completions_ module as shown in the [Example configuration](https://github.com/zimfw/zimfw#example-configuration). So, if you are using that module then remove any call to `compinit` in your `.zshrc` like:
+```sh
+autoload -Uz compinit
+compinit
+# End of lines added by compinstall
+```
+which maybe present after fresh install of zsh or anywhere in your system-wide zsh files like `/etc/zsh/zshrc` which comes with some installations using distro package manager. Otherwise, add:
+```sh
+autoload -Uz compinit
+compinit
+```
+before `zmodule ...` calls in your `.zshrc`
```

Comment:
`zmodule` calls should always be in the .zimrc file. Any code you want to run during the shell startup, should go to the other dotfiles, because the .zimrc file is not sourced during startup. This is common confusion with new Zim users. See https://github.com/zimfw/zimfw/issues/448.

Reply:
Could you answer the 4th _FAQ_ ? I placed the answer as a placeholder. I actually wanted to know your opinion on it. Also, could you take a look at the example configuration file given.